### PR TITLE
Extra validation checking if ticket was opened per logs

### PR DESCRIPTION
**Pull request by oshercc**

### CODE REVIEW

File: tools/create_triage_tickets.py [hunk 1]
```diff
@@ -169,6 +199,8 @@ def main(arg):
              '{signature_type: {root_issue: message}}',
         default='./triage_resolving_filters.json',
     )
+    parser.add_argument("--mark-root", default=None, required=False, help="Location of the logs for marking as "
+                                                                             "handheld")
```
Comment (osherdp) [inline tools/create_triage_tickets.py:203]:
```suggestion
    parser.add_argument("--mark-root", default=None, required=False, 
                        help="Location of the logs for marking as handheld")
```

Comment (osherdp) [reply tools/create_triage_tickets.py:203]: *handled

File: tools/create_triage_tickets.py [hunk 2]
```diff
@@ -72,6 +73,31 @@ def add_watchers(jclient, issue):
     for watcher in DEFAULT_WATCHERS:
         jclient.add_watcher(issue.key, watcher)
 
+
+def is_logfile_handled(existing_tickets, failure_id):
+
```
Comment (osherdp) [inline tools/create_triage_tickets.py:78]: delete

File: tools/create_triage_tickets.py [hunk 3]
```diff
@@ -72,6 +73,31 @@ def add_watchers(jclient, issue):
...
+    marked_path = os.path.join(args.mark_root, failure_id, MARK_ID)
+    if os.path.isfile(marked_path):
+        logger.info(f"{failure_id} is marked as handled")
+        return True
+
+    # if not marked as handled check if has ticket
+    summary = format_summary({"failure_id": failure_id})
+    if summary in existing_tickets:
+        logger.info(f"Marking {failure_id} as handled due, fount summary {summary}")
```
Comment (osherdp) [inline tools/create_triage_tickets.py:87]: typo, and reformat sentence as it's not clear imho

File: tools/create_triage_tickets.py [hunk 4]
```diff
@@ -72,6 +73,31 @@ def add_watchers(jclient, issue):
...
+        mark_logs_as_handled(marked_path, summary)
+        return True
+
+    logger.info(f"Item {existing_tickets} looks new")
```
Comment (osherdp) [inline tools/create_triage_tickets.py:91]:
```suggestion
    logger.info(f"Item {summary} looks new")
```

**Review by osherdp — CHANGES_REQUESTED** [review]

File: tools/create_triage_tickets.py [hunk 5]
```diff
@@ -169,6 +199,8 @@ def main(arg):
              '{signature_type: {root_issue: message}}',
         default='./triage_resolving_filters.json',
     )
+    parser.add_argument("--mark-root", default=None, required=False, help="Location of the logs for marking as "
```
Comment (osherdp) [inline tools/create_triage_tickets.py:202]: not sure about the name, but also I don't have any other suggestion than ``mark-root``
@YuviGold wdyt?

**Review by osherdp — CHANGES_REQUESTED** [review]

File: tools/create_triage_tickets.py [hunk 6]
```diff
@@ -135,6 +161,10 @@ def main(arg):
         cluster = res.json()['cluster']
 
         if cluster['status'] == "error":
+            if args.log_marking_root and is_logfile_handled(summaries, failure['name']):
+                logger.info("issue found: %s", failure['name'])
+                continue
+
```
Comment (omertuc) [inline tools/create_triage_tickets.py:167]: After creating ticket should call mark_as_handled

**Comment by YuviGold** [comment]
@oshercc is this PR still relevant? Please add a description to the PR if it is.

**Pull request closed without merging**